src/templates/shared.js

```diff
@@ -83,8 +83,8 @@ const metabox = ({metaTitle, metaColour, img, imgCaption, mdSections, mdFooter,
   `;
 };
 
-const renderMarkdown = (md, mdFooter) => {
-  return mdRenderer.render(mdFooter ? (md + "\n\n" + mdFooter) : md);
+const renderMarkdown = (md, metaIndex) => {
+  return mdRenderer.render(metaIndex ? (md + "\n\n" + metaIndex.mdFooter) : md);
```

Comment:
Giving the markdown renderer access to the whole `metaIndex` to ease future code changes. This area will may need some work to support anchor links in cross-page references (e.g. `[model tag structure][gbxmodel#structure]`). I may do that by accessing `metaIndex.pages` or modify `metaIndex.mdFooter` upstream.